zcash_client_sqlite/src/lib.rs

```diff
@@ -107,12 +102,21 @@ impl fmt::Display for NoteId {
 pub struct UtxoId(pub i64);
 
 /// A wrapper for the SQLite connection to the wallet database.
-pub struct WalletDb<P> {
-    conn: Connection,
+pub struct WalletDb<C, P> {
+    conn: C,
     params: P,
 }
 
-impl<P: consensus::Parameters> WalletDb<P> {
+/// A wrapper for a SQLite transaction affecting the wallet database.
+pub struct WalletTransaction<'conn>(pub(crate) rusqlite::Transaction<'conn>);
```

Comment:
This name is ambiguous; maybe `WalletSqlTransaction`?

(`WalletDbTransaction` would still be ambiguous since it could be read as `(WalletDb)Transaction`, not the intended `Wallet(DbTransaction)`.)

Reply:
Fixed in https://github.com/zcash/librustzcash/pull/857/commits/8ebda21655755d8dcfe49e4abe09adf7f9659d34